spec/core/finalization/schemas/MessageGroup.yml

```diff
@@ -5,6 +5,9 @@ required:
   - hashes
   - signatures
 properties:
+  signatureSchema:
+    description: Signature schema.
```

Comment:
Not a very useful description, specially when the type is an integer. How is the user supposed to know what to do with this number?

Reply:
SignatureSchema is removed in public net (main branch).

Reply:
only returned from server, user won't need to provide it